blocks-common/i-bem/__dom/i-bem__dom.js

```diff
@@ -1715,3 +1715,35 @@ $(function() {
 });
 
 })(BEM, jQuery);
+
+
+/**
+ * Support for YModules
+ */
+if (typeof modules === 'object') {
+
+    modules.define(
+        'i-bem__dom',
+        function(provide) {
+            provide(BEM.DOM);
+        }
+    );
+
+    // Realization from bem-core#2.5.0
+
+    (function() {
+
+        var origDefine = modules.define;
+
+        modules.define = function(name, deps, decl) {
+            origDefine.apply(modules, arguments);
+
+            name !== 'i-bem__dom_init' && arguments.length > 2 && ~deps.indexOf('i-bem__dom') &&
```

Comment:
> I think it's not a good idea just because it's a 1-to-1 copy from bem-core. If these lines will change in bem-core it will be easier to move diff here.

@zxqfox The rule is simple: this code is harder to understand for the ordinal reader than simple `if` block. So it should be rewritten.

I don't support some of the decisions made for `bem-core` code style. In my opinion, they are for robots, not for humans.